CHANGELOG.md

```diff
@@ -17,14 +17,20 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased](https://github.com/o1-labs/o1js/compare/3eef10d...HEAD)
 
+### Added
+
+- Added `ZkProgram.analyzeSingleMethod(methodName: string)` to analyze a single method of a ZkProgram. https://github.com/o1-labs/o1js/pull/2217
+  - This is an addition to `ZkProgram.analyzeMethods()` which analyzes all methods of a ZkProgram by executing them.
+  - Now only a single method is analyzed at a time.
+
 ## [2.6.0](https://github.com/o1-labs/o1js/compare/4e23a60...3eef10d) - 2025-05-30
 
 ### Added
 
-- [PR! 1905](https://github.com/o1-labs/o1js/pull/1905) API support for circuit chunking
```

Comment:
Oh I thought we were following that format from now on (PR + description)

Reply:
We have been using the other format since we started adding PRs to the changelog. I am okay with both formats, i just went to path of least resistance to make everything use the same format :D